examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/main/appinfo/AppInfoScreen.kt

```diff
@@ -92,13 +57,6 @@ fun AppInfoScreen(
         val currentApiKeyDescription by remember { derivedStateOf { state.apiKeyDescription } }
         Text(text = "Current user ID: $currentUserID")
         Text(text = "Current API key: $currentApiKeyDescription")
-        lastCustomAction?.let { action ->
-            Text(
-                text = "Last custom action: $action",
-                style = MaterialTheme.typography.bodySmall,
-                color = MaterialTheme.colorScheme.primary,
-            )
-        }
```

Comment:
I asked @facumenzella and he's ok with removing this. We have a Toast already and its mainly for debugging purposes